client/state/login/test/selectors.js

```diff
@@ -83,13 +88,13 @@ describe( 'selectors', () => {
 				'sms'
 			);
 
-			expect( nonce ).to.equal( 'abcdef123456' );
+			expect( nonce ).toStrictEqual( 'abcdef123456' );
 		} );
 	} );
 
 	describe( 'isRequestingTwoFactorAuth', () => {
 		test( 'should return false by default', () => {
-			expect( isRequestingTwoFactorAuth( undefined ) ).to.be.false;
+			expect( isRequestingTwoFactorAuth( EMPTY_STATE ) ).toBeFalsy();
```

Comment:
`.toBeFalsy()` also accepts `undefined`, `null` or `0`. We can be more strict here and test explicitly for a boolean `.toBe( false )`. That applies to most (if not all) other usages of `.toBe{Falsy,Truthy}` in this test, too.

Reply:
Good point, we probably do want to be checking for the stricter values, because reading the Chai docs that's what the previous code did.

This helped find a bug in the `isFormDisabled` reducer, which initialised as `null` instead of `false`!